racetrack_client/README.md

```diff
@@ -1,30 +1,93 @@
-# Racetrack client context
-`racetrack-client` is a CLI client tool for deploying workloads to Racetrack.
+# Racetrack CLI client
+`racetrack-client` is a CLI client tool for deploying and managing workloads in Racetrack.
 
 Racetrack system allows to deploy jobs in a one step.
 It transforms your code to in-operation workloads, e.g. Kubernetes workloads.
 You write some code according to a set of coventions, you include the manifest file which explains the code, 
 and you submit it to Racetrack. A short while after, the service calling your code is in operation.
 
-# Installation
+## Quickstart
+```shell
+# Install racetrack-client
+pip3 install racetrack-client
+
+# Set current remote
+racetrack set remote https://racetrack.platform.example.com
+
+# Log in
+racetrack login T0k3n.g0es.H3r3
+
+# Deploy a Job
+racetrack deploy
+```
+
+## Installation
 Install racetrack-client using pip:
-```bash
+```shell
 pip3 install racetrack-client
 ```
 
-Python 3.8+ is required. So if you have any troubles, try with:
+Python 3.8 (or higher) is required.
+
+This will install `racetrack` CLI tool. Verify installation by running `racetrack` command.
+
+## Usage
+Run `racetrack --help` to see usage.
+
+### Adding a remote
+Assuming your Racetrack server is running on https://racetrack.platform.example.com/lifecycle,
+you can add this remote as an alias:
+```shell
+racetrack set alias my-dev https://racetrack.platform.example.com/lifecycle
 ```
-sudo apt install python3.8 python3.8-dev python3.8-venv
-python3.8 -m pip install racetrack-client
+
+### Switching remotes
+Set your current remote with:
+```shell
+racetrack set remote my-dev
 ```
+This will set up a "remote" context for later use.
 
-This will install `racetrack` CLI tool. Verify installation by running `racetrack`.
+### Logging in
+Log in to Racetrack with your user account (you can get your token from the Dashboard's profile page):
+```shell
+racetrack login T0k3n.g0es.H3r3
+```
 
-# Usage
-Run `racetrack --help` to see usage.
+In case you're going to use a private repository, provide your git credentials so the job can be built from your code:
+```shell
+racetrack set credentials https://github.com/YourUser/YourRepository USERNAME TOKEN
+```
+
+### Installing plugins
+Extend Racetrack's possibilities by installing a bunch of plugins:
+```shell
+racetrack plugin install github.com/TheRacetrack/plugin-python-job-type
```

Comment:
It's entirely possible that J. Random User just discovered this project and has only seen this doc, and is executing these steps. Short comments above each explaining why we're installing two and how they're different?